config/local.yaml

```diff
@@ -43,6 +43,7 @@ reporters:
     host: "0.0.0.0:8125"
     prefix: "maestro."
     region: "region"
+    restartTimeout: 30m
```

Comment:
In my opinion, this should be 1 minute, maybe 5 minutes at most. Otherwise, we'll be 30 minutes without metrics and the problem will be perceptive by the user anyway.

Reply:
Changed to 1 minute!